com.playeveryware.eos/Runtime/Core/Config/Attributes/ConfigFieldAttribute.cs

```diff
@@ -60,5 +67,13 @@ public ConfigFieldAttribute(
             Group = group;
             FieldType = type;
         }
+
+        public ConfigFieldAttribute(string label,
+            Enum enumType,
+            string tooltip = null,
+            int group = -1) : this(label, ConfigFieldType.Enum, tooltip, group)
+        {
+            EnumType = enumType;
+        }
```

Comment:
In keeping with earlier feedback regarding not including a field member to keep track of the Enum, this constructor should be removed.